### PR TITLE
[QoL][Refactor] replace `-1` with `RELEARN_MOVE` and `0` with `EVOLVE_MOVE`

### PR DESCRIPTION
**Pull request by flx-sta**
## What are the changes?

replaced magic numbers with descriptive constansts in `pokemon-level-moves.ts`

## Why am I doing these changes?

makes it more clear what the numbers are user for
image

## How to test the changes?

if it builds & runs all good

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I tested the changes (manually)?
    - [ ] Are all unit tests still passing? (`npm run test`)

### CODE REVIEW

**Review by Tempo-anon — DISMISSED** [review]
Should add a comment to these `const`s

**Comment by flx-sta** [comment]
> Should add a comment to these `const`s

Will do

**Comment by flx-sta** [comment]
@Tempo-anon Done

File: src/data/pokemon-level-moves.ts [hunk 1]
```diff
@@ -15,6 +15,11 @@ interface PokemonSpeciesFormLevelMoves {
   [key: integer]: PokemonFormLevelMoves
 }
 
+/** Moves that can only be learned with a memory-🍄 */
```
Comment (Tempo-anon) [inline src/data/pokemon-level-moves.ts:18]: It's funny but let's not start adding emojis into files

Comment (flx-sta) [reply src/data/pokemon-level-moves.ts:18]:
```suggestion
/** Moves that can only be learned with a memory-mushroom */
```

**Review by Tempo-anon — APPROVED** [review]

**Pull request merged**